app/utils/APIUtils.js

```diff
@@ -64,15 +64,16 @@ function getHeadersCreator(headers) {
 }
 
 function getRequestTypeDescriptor(type, options = {}) {
+  const meta = options.meta || {};
```

Comment:
- [ ] This is actually not needed. `Object.assign` will work even if options.meta is `undefined` or `null`.